content/v2.md

```diff
@@ -193,6 +193,37 @@ were created in and notices based on these objects will be delivered with
 the proper locale. i.e. if an issue is created with the "es" locale, all
 email to the creator will be delivered with Spanish translations.
 
+## Terms of Use
+
+### API Rate Limits
+
+The rate limit allows for up to 20 requests per minute. 
+
+Requests are associated with the originating IP address.
+
+Rate limit is subject to change.
+
+#### HTTPS Headers and Response Codes
+
+If you go over the rate limits, we will return a HTTP 429 Too Many Requests error, and a Retry-After HTTP header containing the number of seconds until you can retry.
+
+```
+{
+    "message": "Too Many Requests",
+    "documentation_url": "http://dev.seeclickfix.com"
+}
+```
+
+```
+ Retry-After 10
+```
+
+#### Blacklisting
+
+If an application abuses the rate limits, it will be blacklisted. If you or your application has been blacklisted and you think there has been a mistake, please contact us at <support@seeclickfix.com>
+
```

Comment:
Should mention that 403 errors will be returned in that case.

Reply:
Added 403 text